src/main/resources/templates/fragments/userBar.html

```diff
@@ -4,7 +4,7 @@
         <title>User Bar</title>
     </head>
     <body>
-        <div th:fragment="userBar" class="js-toggle-nav" id="global-nav">
+        <div th:fragment="userBar" th:classappend="${hideUserBar != null ? 'hidden' : ''}" class="js-toggle-nav" id="global-nav">
```

Comment:
Just a minor thought, rather than passing a new attribute to the fragment, could we conditionally display the user bar based on whether `userEmail` is null?